2-copy-of-code/lesson-13/scripts/amazon.js

```diff
@@ -72,6 +72,7 @@ document.querySelectorAll('.js-add-to-cart')
       const quantitySelector = document.querySelector(
         `.js-quantity-selector-${productId}`
```

Comment:
amazon.js:77 Uncaught ReferenceError: productId is not defined
    at HTMLButtonElement.<anonymous> (amazon.js:77:83) 
    
    Getting this error while trying to select <select> element

Reply:
[connect me on Instagram](https://www.instagram.com/banda.shri/)